### PR TITLE
Added an old value parameter for Bindable.ValueChange

### PR DESCRIPTION
**Pull request by MrTheMake**

### CODE REVIEW

**Comment by peppy** [comment]
Do you have an osu!-side PR where this change is actually used?

**Comment by MrTheMake** [comment]
I'm planning on eventually fixing https://github.com/ppy/osu/issues/951.

**Comment by smoogipoo** [comment]
I'm not merging this without an osu!-size PR since this is 100% a breaking change that needs appropriate changes to compile.

**Comment by peppy** [comment]
Also it needs use cases that show how it can actually reduce code complexity

File: osu.Framework.Tests/Visual/TestCaseBlending.cs [hunk 1]
```diff
@@ -146,9 +146,9 @@ public TestCaseBlending()
             colourEquation.Current.Value = foregroundContainer.Blending.RGBEquation;
             alphaEquation.Current.Value = foregroundContainer.Blending.AlphaEquation;
 
-            colourModeDropdown.Current.ValueChanged += v => updateBlending();
-            colourEquation.Current.ValueChanged += v => updateBlending();
-            alphaEquation.Current.ValueChanged += v => updateBlending();
+            colourModeDropdown.Current.ValueChanged += (newValue, oldValue) => updateBlending();
```
Comment (Tom94) [inline osu.Framework.Tests/Visual/TestCaseBlending.cs]: For code cleanliness I would recommend naming unused parameters _, __, and so on. That makes clear the parameters are unused. Likewise, if the old value is not used, then the old usage of `v => foo(v)` would become `(v, _) => foo(v)` which imho is not a huge deal in terms of complexity.

**Comment by smoogipoo** [comment]
Closing due to lack of activity. In general, I'm fine with this, but I want to see an osu!-side PR, as previously asked.

**Pull request closed without merging**